src/js/containers/covid19/awardSpendingAgency/AwardSpendingAgencyTableContainer.jsx

```diff
@@ -293,12 +308,6 @@ const AwardSpendingAgencyTableContainer = (props) => {
         window.scrollTo(0, 0);
     }, [document]);
 
-    useEffect(() => {
```

Comment:
We actually do need to keep this. A `useEffect` with an empty dependency array is equivalent to like a `componentDidMount` and the cleanUp fn for it is like `componentWillUnumount`. So actually we can probably just keep this here and delete the cancels/clean up functions elsewhere that you've added.